src/generate-lo-mod-rules.ts

```diff
@@ -0,0 +1,74 @@
+import { getAll, loadLocal } from '@d2api/manifest-node';
+import { ItemCategoryHashes } from '../data/generated-enums.js';
+import { writeFile } from './helpers.js';
+
+loadLocal();
+const loModRules: Record<string, string[]> = {};
+
+const debug = true;
+
+const allArmorMods = getAll('DestinyInventoryItemDefinition').filter((i) =>
+  i.itemCategoryHashes?.includes(ItemCategoryHashes.ArmorMods)
+);
+
+const allSandboxPerks = getAll('DestinySandboxPerkDefinition');
+
+const allArmorModsDesc: Record<string, string> = {};
+
+allArmorMods.forEach((i) => {
+  loModRules[i.hash] = [];
+  const armorModName = i.displayProperties.name;
+  const itemWithSameName = allSandboxPerks.find((p) => p.displayProperties.name === armorModName);
+  // currently matching on description instead of i.perks[x] lookup
+  if (itemWithSameName?.displayProperties.description) {
+    allArmorModsDesc[i.hash] = itemWithSameName.displayProperties.description;
+  }
+  if (armorModName.match(/radiant light|powerful friends/i)) {
+    loModRules[i.hash].push('max.1');
+  }
+  if (armorModName.match(/finisher/i)) {
+    loModRules[i.hash].push('finisher');
```

Comment:
for exclusives it might be nice to do something like the `uniqueStackLabel` - a string that identifies "you can't have more than one thing with this string"